src/test/regress/expected/multi_prepare_sql.out

```diff
@@ -847,9 +843,7 @@ SELECT * FROM prepare_table ORDER BY key, value;
    0 |      
    0 |      
    0 |      
-   6 |    60
-   6 |      
-(8 rows)
+(6 rows)
```

Comment:
While trying to understand why this table changed on this file without any other update, I noticed that there is a copy/paste mistake here and we use a table from another regression test. I'm pushing a commit to fix it.